File: wa-backend/internal/model/template.go
package model

import (
	"encoding/json"
	"time"
)

type Template struct {
	ID             string          `json:"id"`
	Name           string          `json:"name"`
	Category       string          `json:"category"`
	Language       string          `json:"language"`
	Status         string          `json:"status"`
	Components     json.RawMessage `json:"components"`
	MetaTemplateID *string         `json:"meta_template_id,omitempty"`
	CreatedAt      time.Time       `json:"created_at"`
	UpdatedAt      time.Time       `json:"updated_at"`
}

File: wa-desktop/src/WaDesktop.Domain/Entities/Template.cs
namespace WaDesktop.Domain.Entities
{
    public class Template
    {
        public string Id { get; set; }
        public string WabaId { get; set; }
        public string Name { get; set; }
        public string Language { get; set; }
        public string Status { get; set; }
        public string Category { get; set; }
        public int? MessageSendTtlSeconds { get; set; }
        public string ParameterFormat { get; set; }
    }
}
